tooling-ssl-operations/pom.xml

```diff
@@ -35,5 +35,15 @@
             <artifactId>wildfly-cli</artifactId>
         </dependency>
     </dependencies>
-
+    <build>
```

Comment:
What is the reason for skipping tests in tooling modules?

Reply:
This module doesn't contain any test code, I was getting Maven errors that could be ommited by `-DfailIfNoTests=false` but this approach seemed cleaner (similarly as I've done in other moodules that don't contain any test code)

Reply:
@spriadka ^^^